aws/logs_monitoring/README.md

```diff
@@ -100,15 +100,16 @@ If you can't install the Forwarder using the provided CloudFormation template, y
 
 ## AWS PrivateLink Support
 
-Configure the Forwarder using AWS PrivateLink to run in a VPC.
+You can run the Forwarder in a VPC by using AWS PrivateLink.
 
-1. Follow the [setup instructions](https://docs.datadoghq.com/agent/guide/private-link/?tab=logs#create-your-vpc-endpoint) for adding Datadog's endpoints to your VPC.
+1. Follow the [setup instructions](https://docs.datadoghq.com/agent/guide/private-link/?tab=logs#create-your-vpc-endpoint) for adding Datadog's endpoints to your VPC. **Be sure to add endpoints for both the Datadog API and for Logs.** 
```

Comment:
Let's actually break this into two steps or two sub steps? It's probably more effective than adding a note.